Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CanvasPathGeometry/CanvasPathGeometryPage.xaml.cs

```diff
@@ -0,0 +1,253 @@
+            "114.483,17.106 ZM 140.431,32.645 C 140.431,33.192 140.225,33.655 139.815,34.034 C 139.405,34.414 138.838,34.603 138.118,34.603 C 137.396,34.603 136.830,34.414 136.420,34.034 C " +
+            "136.010,33.655 135.804,33.192 135.804,32.645 C 135.804,32.110 136.006,31.653 136.411,31.274 C 136.815,30.895 137.384,30.705 138.118,30.705 C 138.851,30.705 139.420,30.891 " +
+            "139.824,31.264 C 140.228,31.637 140.431,32.098 140.431,32.645 Z M 141.046,13.655 L 139.983,25.183 C 139.933,25.780 139.734,26.244 139.386,26.573 C 139.038,26.903 138.603,27.067 " +
+            "138.080,27.067 C 137.558,27.067 137.123,26.903 136.774,26.573 C 136.426,26.244 136.227,25.780 136.178,25.183 L 135.096,13.655 C 135.046,13.071 135.021,12.685 135.021,12.499 C " +
+            "135.021,11.529 135.313,10.749 135.898,10.158 C 136.482,9.567 137.210,9.272 138.080,9.272 C 138.938,9.272 139.662,9.570 140.253,10.167 C 140.844,10.764 141.139,11.516 141.139,12.424 " +
+            "C 141.139,12.611 141.108,13.021 141.046,13.655 Z";
+
+        private List<Color> _colors;
+        private List<string> _samples;
+
+        private string _data = string.Empty;
+        private StringBuilder _logger;
+
+        private float _strokeThickness;
+        private Color _strokeColor;
+        private Color _fillColor;
+        private bool _selectionChanged = false;
+        private bool _isParsing = false;
+
+        private CanvasGeometry _errorGeometry;
+
+        public string InputText { get; set; }
+
+        public CanvasPathGeometryPage()
+        {
+            this.InitializeComponent();
+            _logger = new StringBuilder();
+            _colors = new List<Color>()
+            {
+                Colors.Transparent,
+                Colors.Black,
+                Colors.White,
+                Colors.Crimson,
+                CanvasPathGeometry.CreateColor("#bf5af2"),
+                CanvasPathGeometry.CreateColor("#0a84ff"),
+                CanvasPathGeometry.CreateColor("#32d74b"),
+                CanvasPathGeometry.CreateColor("#ff9500"),
+                CanvasPathGeometry.CreateColor("#ffd60a")
+            };
+
+            var colorList = new List<string>()
+            {
+                "Transparent",
+                "Black",
+                "White",
+                "Crimson",
+                "Purple",
+                "LightBlue",
+                "LightGreen",
+                "Orange",
+                "Yellow"
+            };
+
+            this._samples = new List<string>()
+            {
+                string.Empty,
+                Sample1,
+                Sample2,
+                Sample3,
+                Sample4,
+                Sample5
+            };
+
+            var sampleList = new List<string>()
+            {
+                "None",
+                "Sample 1",
```

Comment:
Can we have these named more descriptively based on the shape they draw?

Reply:
Renamed and added the samples as buttons.